core/src/main/java/org/eclipse/hono/util/EventConstants.java

```diff
@@ -29,4 +29,15 @@
 
     private EventConstants() {
     }
+
+    /**
+     * The content type that is defined for empty events without any payload.
+     */
+    public static final String CONTENT_TYPE_EMPTY_NOTIFICATION = "application/vnd.eclipse-hono-empty-notification";
+
+    /**
+     * The content type that is defined for device connection notification events.
+     */
+    public static final String CONTENT_TYPE_DEVICE_CONNECTION_NOTIFICATION = "application/vnd.eclipse-hono-dc-notification+json";
```

Comment:
is this relevant for devices indicating their readiness to receive a command?

Reply:
No, it was meant as minimal step to integrate with the `connected/disconnected` event.

I will eliminate it for clarity and we introduce it later.